sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch

```diff
@@ -46,6 +46,7 @@
   <!-- This will include the wrist joints in the hand controller and exclude them from the arm one -->
   <arg name="include_wrist_in_arm_controller" default="true" if="$(arg arm_ctrl)"/>
   <arg name="include_wrist_in_arm_controller" default="false" unless="$(arg arm_ctrl)"/>
+  <arg name="urcap_program_name" default="external_ctrl.urp"/>
```

Comment:
lets set this to correct value depending it robot_model is ur10 or ur10e

Reply:
Do you know the name of the program on the ur10e? I think it's arbitrary, the person who initially sets the program up can name it whatever they like. Think we should re-name it on the arm instead?

Reply:
please discuss this with beatriz

Reply:
@beatrizleon

Reply:
if we don't know the correct value for the ur10e, then we can leave it like this and we can refactoring later